libparsec/crates/protocol/tests/authenticated_cmds/v4/invite_greeter_cancel_greeting_attempt.rs

```diff
@@ -0,0 +1,130 @@
+
+pub fn rep_invitation_cancelled() {
+    // Generated from Parsec 3.0.0-b.12+dev
+    // Content:
+    //   status: 'invitation_cancelled'
+    let raw: &[u8] = hex!("81a6737461747573b4696e7669746174696f6e5f63616e63656c6c6564").as_ref();
+    let expected = authenticated_cmds::invite_greeter_cancel_greeting_attempt::InviteGreeterCancelGreetingAttemptRep::InvitationCancelled;
+    println!("***expected: {:?}", expected.dump().unwrap());
+    let data = authenticated_cmds::invite_greeter_cancel_greeting_attempt::Rep::load(raw).unwrap();
+    p_assert_eq!(data, expected);
+}
+
+pub fn rep_invitation_completed() {
+    // Generated from Parsec 3.0.0-b.12+dev
+    // Content:
+    //   status: 'invitation_completed'
+    let raw: &[u8] = hex!("81a6737461747573b4696e7669746174696f6e5f636f6d706c65746564").as_ref();
+    let expected = authenticated_cmds::invite_greeter_cancel_greeting_attempt::InviteGreeterCancelGreetingAttemptRep::InvitationCompleted;
+    println!("***expected: {:?}", expected.dump().unwrap());
+    let data = authenticated_cmds::invite_greeter_cancel_greeting_attempt::Rep::load(raw).unwrap();
+    p_assert_eq!(data, expected);
+}
+
+pub fn rep_greeting_attempt_not_found() {
+    // Generated from Parsec 3.0.0-b.12+dev
+    // Content:
+    //   status: 'greeting_attempt_not_found'
+    let raw: &[u8] =
+        hex!("81a6737461747573ba6772656574696e675f617474656d70745f6e6f745f666f756e64").as_ref();
+    let expected = authenticated_cmds::invite_greeter_cancel_greeting_attempt::InviteGreeterCancelGreetingAttemptRep::GreetingAttemptNotFound;
+    println!("***expected: {:?}", expected.dump().unwrap());
+    let data = authenticated_cmds::invite_greeter_cancel_greeting_attempt::Rep::load(raw).unwrap();
+    p_assert_eq!(data, expected);
+}
+
+pub fn rep_greeting_attempt_not_joined() {
+    // Generated from Parsec 3.0.0-b.12+dev
+    // Content:
+    //   status: 'greeting_attempt_not_joined'
+    let raw: &[u8] = hex!(
+    "81a6737461747573bb6772656574696e675f617474656d70745f6e6f745f6a6f696e65"
+    "64"
+    )
+    .as_ref();
+    let expected = authenticated_cmds::invite_greeter_cancel_greeting_attempt::InviteGreeterCancelGreetingAttemptRep::GreetingAttemptNotJoined;
+    println!("***expected: {:?}", expected.dump().unwrap());
+    let data = authenticated_cmds::invite_greeter_cancel_greeting_attempt::Rep::load(raw).unwrap();
+    p_assert_eq!(data, expected);
+}
+
+pub fn rep_greeting_attempt_already_cancelled() {
+    // Generated from Parsec 3.0.0-b.12+dev
+    // Content:
+    //   status: 'greeting_attempt_already_cancelled'
+    //   origin: 'GREETER'
+    //   reason: 'MANUALLY_CANCELLED'
+    //   timestamp: ext(1, 946774800000000) i.e. 2000-01-02T02:00:00Z
+    let raw: &[u8] = hex!(
+    "84a6737461747573d9226772656574696e675f617474656d70745f616c72656164795f"
+    "63616e63656c6c6564a66f726967696ea747524545544552a6726561736f6eb24d414e"
+    "55414c4c595f43414e43454c4c4544a974696d657374616d70d70100035d162fa2e400"
+    )
```

Comment:
```suggestion
    let raw: &[u8] = hex!(
        "84a6737461747573d9226772656574696e675f617474656d70745f616c72656164795f"
        "63616e63656c6c6564a66f726967696ea747524545544552a6726561736f6eb24d414e"
        "55414c4c595f43414e43454c4c4544a974696d657374616d70d70100035d162fa2e400"
    )
```

Indentation seems off, `rustfmt` did not format the tests ?

Reply:
Rustfmt doesn't always interact well with macros :grimacing: